phpunit.xml.dist

```diff
@@ -9,17 +9,20 @@
          processIsolation="false"
          stopOnFailure="false"
          syntaxCheck="false"
-         bootstrap="tests/bootstrap.php"
+         bootstrap="vendor/symfony-cmf/testing/bootstrap/bootstrap.php"
 >
     <php>
-        <!-- Disable E_USER_DEPRECATED until 3.0 -->
-        <!-- php -r 'echo -1 & ~E_USER_DEPRECATED;' -->
```

Comment:
https://github.com/sonata-project/SonataDoctrinePhpcrAdminBundle/commit/c20f2e88a5439976f6d652cba3465f507ea22e88 should be reverted IMO